bedrock/settings/static_media.py

```diff
@@ -1013,6 +1013,18 @@
         ),
         'output_filename': 'js/common-bundle.js',
     },
+    'common-oldie': {
+        'source_filenames': (
+            'js/libs/jquery-1.11.3.min.js',  # upgrade to jQuery 1.12?
+            # will need to make sure scripts below are compatible with jQuery 1.12 & 3?
```

Comment:
This is the only thing I think we'll need to be careful about. I do wonder if we can cut this down even further maybe. If we're showing dumb download buttons to old IE, could we theoretically cut out `global.js, `global.init.js` and `mozilla-client.js`?